tests/Aspire.Workload.Tests/StarterTemplateProjectNamesTests.cs

```diff
@@ -1,50 +1,78 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common.Tests;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Aspire.Workload.Tests;
 
-public class StarterTemplateProjectNamesTests : WorkloadTestsBase
+public abstract class StarterTemplateProjectNamesTests : WorkloadTestsBase
 {
-    public StarterTemplateProjectNamesTests(ITestOutputHelper testOutput)
+    private readonly string _testType;
+    public StarterTemplateProjectNamesTests(string testType, ITestOutputHelper testOutput)
         : base(testOutput)
     {
+        _testType = testType;
     }
 
-    public static TheoryData<string, string> ProjectNamesWithTestType_TestData()
+    public static IEnumerable<object[]> ProjectNamesWithTestType_TestData()
```

Comment:
Why the move away from TheoryData?

Reply:
Err leftover from a different change. Thank you, fixing it now!